ENV_VARS.md

```diff
@@ -12,7 +12,7 @@ Agents can be configured using environment variables:
 | HT_REPORTING_SECURE | When `true`, connects to endpoints over TLS. |
 | HT_REPORTING_TOKEN | User specific token to access Traceable API |
 | HT_REPORTING_CERT_FILE | Certificate file containing the CA to verify the server's certificate. This is for private certificates. If this is set then `secure` above should also be set to `true`. |
```

Comment:
Should we add env var entries in this file for `HT_REPORTING_TRACE_REPORTER_TYPE` and `HT_REPORTING_METRIC_REPORTER_TYPE`?

Reply:
We will need to fix the env var generation script. I dont know why these did not get generated. We should not modify this file manually.

Reply:
ah, gotcha. In that case, this is out of scope